recipes/sigprofilerplotting/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "SigProfilerPlotting" %}
+{% set version = "1.3.21" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sigProfilerPlotting-{{ version }}.tar.gz
+  sha256: 2b7805adc9d00c275bc7cb9af16a732ea3d7cccea68dc8ea3f96b4cd2a405550
+
+build:
+  entry_points:
+    - SigProfilerPlotting=sigProfilerPlotting.sigProfilerPlotting_CLI:main_function
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name|lower, max_pin="x") }}
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - matplotlib-base >=3.4.3
+    - pandas >=1.2.4,<2.0.0
+    - scikit-learn >=1.1.3
+    - pillow >=10.0.0
+
+test:
+  imports:
+    - sigProfilerPlotting
+  commands:
+    - pip check
+    - SigProfilerPlotting plotSBS --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/alexandrovlab/SigProfilerPlotting
+  summary: SigProfiler plotting tool
+  license: BSD-2-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - xAranaktu
```

Comment:
Was the user automatically added, or did they agree to be a maintainer?

Reply:
He was added automatically by the grayskull. Remove?

Reply:
What about the license? Is it enough that the license type is just mentioned in this yaml?

Reply:
yeah let's definitely remove.

If you don't want to maintain that's fine too, just remove the whole maintainers section

Reply:
Not a lawyer, but I think we are OK listing the license. I checked a few other recipes under the same license and the LICENSE file is not repackaged.